packages/apollo-server-integration-testsuite/src/ApolloServer.ts

```diff
@@ -2267,16 +2267,20 @@ export function testApolloServer<AS extends ApolloServerBase>(
 
       it('basic caching', async () => {
         class TTLTestingCache implements KeyValueCache<string> {
+          private fakeTime = 0;
           constructor(
             private cache: Map<
               string,
-              { value: string; ttl: number | null }
+              { value: string; ttl: number | null; fakeTimeOnSet: number }
```

Comment:
i think logic is a bit easier to understand if you store the deadline instead of ttl + time of set, though I don't think this is wrong...

Reply:
Agreed, good suggestion [20a46c5](https://github.com/apollographql/apollo-server/pull/6522/commits/20a46c59e28180abf9b0904b7776a7a92a660b70)